src/topology/vector_bundle/hom.lean

```diff
@@ -0,0 +1,267 @@
+by delta_instance bundle.continuous_linear_map
+
+variables [∀ x, has_continuous_smul 𝕜₂ (E₂ x)]
+
+instance (x : B) : module 𝕜₂ (bundle.continuous_linear_map σ F₁ E₁ F₂ E₂ x) :=
+by delta_instance bundle.continuous_linear_map
+
+end defs
+
+variables {𝕜₁ : Type*} [nondiscrete_normed_field 𝕜₁] {𝕜₂ : Type*} [nondiscrete_normed_field 𝕜₂]
+  (σ : 𝕜₁ →+* 𝕜₂)
+
+variables {B : Type*} [topological_space B]
+
+variables (F₁ : Type*) [normed_group F₁] [normed_space 𝕜₁ F₁]
+  (E₁ : B → Type*) [Π x, add_comm_monoid (E₁ x)] [Π x, module 𝕜₁ (E₁ x)]
+  [topological_space (total_space E₁)]
+variables (F₂ : Type*) [normed_group F₂][normed_space 𝕜₂ F₂]
+  (E₂ : B → Type*) [Π x, add_comm_monoid (E₂ x)] [Π x, module 𝕜₂ (E₂ x)]
+  [topological_space (total_space E₂)]
+
+namespace topological_vector_bundle
+
+variables {F₁ E₁ F₂ E₂} (e₁ e₁' : trivialization 𝕜₁ F₁ E₁) (e₂ e₂' : trivialization 𝕜₂ F₂ E₂)
+variables [ring_hom_isometric σ]
+
+namespace pretrivialization
+
+/-- The coordinate change function between two trivializations of the vector bundle of
+continuous linear maps. -/
+def continuous_linear_map_coord_change (b : B) : (F₁ →SL[σ] F₂) →L[𝕜₂] F₁ →SL[σ] F₂ :=
+((e₁'.coord_change e₁ b).symm.arrow_congrSL (e₂.coord_change e₂' b) :
+  (F₁ →SL[σ] F₂) ≃L[𝕜₂] F₁ →SL[σ] F₂)
+
+
+variables {σ e₁ e₁' e₂ e₂'}
+variables [Π x : B, topological_space (E₁ x)] [topological_vector_bundle 𝕜₁ F₁ E₁]
+variables [Π x : B, topological_space (E₂ x)] [topological_vector_bundle 𝕜₂ F₂ E₂]
+
+lemma continuous_on_continuous_linear_map_coord_change
+  (he₁ : e₁ ∈ trivialization_atlas 𝕜₁ F₁ E₁) (he₁' : e₁' ∈ trivialization_atlas 𝕜₁ F₁ E₁)
+  (he₂ : e₂ ∈ trivialization_atlas 𝕜₂ F₂ E₂) (he₂' : e₂' ∈ trivialization_atlas 𝕜₂ F₂ E₂) :
+  continuous_on (continuous_linear_map_coord_change σ e₁ e₁' e₂ e₂')
+    ((e₁.base_set ∩ e₂.base_set) ∩ (e₁'.base_set ∩ e₂'.base_set)) :=
+begin
+  have h₁ := (compSL F₁ F₂ F₂ σ (ring_hom.id 𝕜₂)).continuous,
+  have h₂ := (continuous_linear_map.flip (compSL F₁ F₁ F₂ (ring_hom.id 𝕜₁) σ)).continuous,
+  have h₃ := (continuous_on_coord_change e₁' he₁' e₁ he₁),
+  have h₄ := (continuous_on_coord_change e₂ he₂ e₂' he₂'),
+  refine ((h₁.comp_continuous_on (h₄.mono _)).clm_comp (h₂.comp_continuous_on (h₃.mono _))).congr _,
+  { mfld_set_tac },
+  { mfld_set_tac },
+  { intros b hb, ext L v,
+    simp only [continuous_linear_map_coord_change, continuous_linear_equiv.coe_coe,
+      continuous_linear_equiv.arrow_congrSL_apply, comp_apply, function.comp, compSL_apply,
+      flip_apply, continuous_linear_equiv.symm_symm] },
+end
+
+variables (σ e₁ e₁' e₂ e₂')
+variables [Π x, has_continuous_add (E₂ x)] [Π x, has_continuous_smul 𝕜₂ (E₂ x)]
+
+/-- Given trivializations `e₁`, `e₂` for vector bundles `E₁`, `E₂` over a base `B`, the induced
+pretrivialization for the continuous `σ`-semilinear maps from `E₁` to `E₂`.  That is, the map which
+will later become a trivialization, after this direct sum is equipped with the right topological
```

Comment:
`direct sum`?

Reply:
Also, can you say "`continuous_linear_map σ e₁ e₂` is the induced pretrivialization..."? Again to help the reader figure out how this is used.